### ISSUE TITLE
Support ".zip" file importing

### ISSUE THREAD

**Issue by operatorequals** [operatorequals/httpimport#8]
It would be great to support `zip`'d Python packages. There is already a built-in functionality with `ZipImporter`, but it needs the `.zip` file to be on disk (listable and with a filename).

Support for `.zip` has to be *in-memory only*.

It is possible that the example code below will be useful:
```
https://stackoverflow.com/questions/39135750/python-load-zip-with-modules-from-memory
```

**Comment by lavvy** [comment]
Any news so far with this feature

**Comment by operatorequals** [comment]
Nop, no implementation yet.
It is gonna happen eventually though.

**Comment by operatorequals** [comment]
Fixed as of `0.5.18`. Now available in PyPi!
